samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/DeliveryBean.java

```diff
@@ -2776,4 +2786,20 @@ public Boolean isTrackingQuestions() {
     public Boolean getTrackingQuestions() {
     	return Boolean.valueOf(publishedAssessment.getAssessmentMetaDataByLabel(AssessmentMetaDataIfc.TRACK_QUESTIONS));
     }
+
+    public String getTimeLimit_bck() {
+      return timeLimit_bck;
+    }
+    
+    public void setTimeLimit_bck(String timeLimit_bck) {
+        this.timeLimit_bck = timeLimit_bck;
+    }
+    
+    public String getTimeElapse_bck() {
+        return timeElapse_bck;
+    }
+    
+    public void setTimeElapse_bck(String timeElapse_bck) {
+        this.timeElapse_bck = timeElapse_bck;
+    }
```

Comment:
if you use lombok you don't need to add the setter and getter

Reply:
also what does bck mean? I would just use the word

Reply:
@ern thank you for your review! I'll correct it and switch to camel case